site/smart-accounts/session-keys/index.md

```diff
@@ -0,0 +1,63 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: Introduction to using Session Keys
+  - - meta
+    - name: description
+      content: Learn about Alchemy's ERC-6900 Compatible Session Key Plugin.
+  - - meta
+    - property: og:description
+      content: Learn about Alchemy's ERC-6900 Compatible Session Key Plugin.
+  - - meta
+    - name: twitter:title
+      content: Introduction to using Session Keys
+  - - meta
+    - name: twitter:description
+      content: Learn about Alchemy's ERC-6900 Compatible Session Key Plugin.
+---
+
+# Session Key Plugin Overview
```

Comment:
should we also mention about the dependencies that the Session Key Plugin has on Multiowner Plugin? cc: @adam-alchemy is there any doc / snippets that explains about this dependency and generally explain about the session keys relationship with the account ownership structure?

Reply:
yea good question. We kinda hide this from devs in the SDK impl, but would be good to highlight I'm sure. @adam-alchemy if you could write up a small suggestion about that here that'd be super helpful